src/Events/Admin/Settings/Filter_Bar_Upsell.php

```diff
@@ -22,6 +22,14 @@
  * @since TBD
  */
 class Filter_Bar_Upsell extends Service_Provider {
+	/**
+	 * The slug of the upsell tab.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $slug = 'filter-view';
```

Comment:
A bit more space around this as well.

```suggestion

	/**
	 * The slug of the upsell tab.
	 *
	 * @since TBD
	 *
	 * @var string
	 */
	protected string $slug = 'filter-view';

```